code/modules/mob/living/simple_animal/simple_animal.dm

```diff
@@ -167,6 +173,26 @@
 
 	return ..()
 
+/mob/living/simple_animal/attackby(obj/item/O, mob/user, params)
+	if(!is_type_in_list(O, food_type))
+		..()
+		return
+	else
+		user.visible_message("<span class='notice'>[user] hand-feeds [O] to [src].</span>", "<span class='notice'>You hand-feed [O] to [src].</span>")
+		qdel(O)
+		if(tame)
+			return
+		else
```

Comment:
Don't nest you early returned for a reason

Reply:
github diff is confused but this should be sorted

Reply:
the diff represents the latest version, you need to psuh

Reply:
I did push my changes, it's just github is now pointing to a different "else" line when you look at this suggestion box in the files changed section and doesn't detect there's been a change